src/content/en/2022/mobile-web.md

```diff
@@ -2,21 +2,436 @@
+
+
+## Mobile performance
+
+One of the most complex problems that site owners have to address on the mobile web is performance, which is experienced as the delay that users incur when they request or interact with websites on smaller devices. Though they have evolved quite a lot, mobile devices still generally have less-powerful processors than larger devices and are often getting data over slower and less reliable internet connections. In addition, the probability that users may incur a cost for mobile data requests, especially if they go over a plan limit, makes efficiency important as a practical concern, beyond just its impact on user experience and speed.
+
+
+### Core Web Vitals
+
+[Core Web Vitals](https://web.dev/vitals/) is a collection of performance metrics that Google compiles to evaluate different websites, and specifically, different [page groups](https://support.google.com/webmasters/answer/9205520#page_groups) on websites to describe how they perform in both mobile and desktop page settings. The elements of Core Web Vitals include loading, interactivity, and layout stability.
+
+All three are aspects of how users perceive the performance of a page that can help or hinder the loading experience for users. This type of evaluation began in May of 2020, and these metrics are taken into account in Google’s ranking algorithm specifically as an aspect of the [page experience](https://developers.google.com/search/docs/advanced/experience/page-experience) evaluation, and thus, the metrics are organized around thresholds of performance that are either considered “good”, “needs improvement”, or “poor”. For a site to be considered “good”, 75% of visits must meet the prescribed thresholds for each of the Core Web Vitals metrics.
+
+{{ figure_markup(
+    image="good-cwv.png",
+    caption="Annual comparison of the percent of websites having good Core Web Vitals on mobile.",
+    description="Bar chart showing 24% of websites having good Core Web Vitals in 2020, 29% in 2021, and 39% in 2022.",
+    chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQ2iu1w8TLfOTLCQ1O-d74ku9NirGCQJNeO789uKX1-M6N053ASsz2tXWPvU1twhScsa3rxWF7_7OBn/pubchart?oid=1388234814&format=interactive",
+    sheets_gid="381713804",
+    sql_file="web_vitals_by_device.sql"
+  )
+}}
+
+The figure above shows a comparison of overall performance for the web in 2022, as it has changed for mobile and desktop measurements since 2020, when Core Web Vitals first launched. You can see that overall, mobile websites are consistently improving year over year. In 2022, 39% of websites have good Core Web Vitals experiences on mobile devices.  See the [Performance](../performance) chapter for a deeper look at what may have caused such a significant change this year.
```

Comment:
```suggestion
The figure above shows how the overall performance of the web has changed since Core Web Vitals first launched in 2020. You can see that overall, mobile websites are consistently improving year over year. In 2022, 39% of websites have good Core Web Vitals experiences on mobile devices.  See the [Performance](../performance) chapter for a deeper look at what may have caused such a significant change this year.
```